### PR TITLE
todd Branch - Frontend now complete.

### PR DESCRIPTION
**Pull request by tt01924**
Frontend now complete. Please merge to the main once you are happy with everything!

### CODE REVIEW

File: .vscode/settings.json
Comment (RGHANILOO) [inline .vscode/settings.json:1]: this file can be added to .gitignore

File: public/Beach1.JPG
Comment (RGHANILOO) [inline public/Beach1.JPG:1]: we can remove all the images, we don't need these

File: public/city.html [hunk 1]
```diff
@@ -11,25 +11,39 @@
     </section>
 
     <section class = "cityRectWide">
-      <h1>City, Country</h1>
-    </section>
-
-    <section class = "cityCircleReturn">
-      <span style='font-size:100px;'>&#8592;</span>
+      <input class = "cityName" placeholder = "City, Country"></input>
```
Comment (RGHANILOO) [inline public/city.html]: not sure if input as an element is a correct choice here 
previously we had H1 here that could potentially work

File: public/city.html [hunk 2]
```diff
@@ -11,25 +11,39 @@
...
     </section>
 
     <section class = "cityCircle">
       <p id="weatherText"></p>
     </section>
 
-    <section class = "cityRectCarousel">
+    <section class = "cityTimeCircle">
+      <input class ="cityTime" placeholder = "Time"></input>
```
Comment (RGHANILOO) [inline public/city.html]: i think input again here is not the best choice perhaps

File: public/city.html [hunk 3]
```diff
@@ -11,25 +11,39 @@
...
-      <h1>City, Country</h1>
-    </section>
-
-    <section class = "cityCircleReturn">
-      <span style='font-size:100px;'>&#8592;</span>
+      <input class = "cityName" placeholder = "City, Country"></input>
     </section>
 
     <section class = "cityCircle">
       <p id="weatherText"></p>
```
Comment (RGHANILOO) [inline public/city.html:18]: is this. single forecast or the forecast of 5 days ?

File: public/city.html [hunk 4]
```diff
@@ -11,25 +11,39 @@
...
     </section>
 
-    <section class = "cityRectCarousel">
+    <section class = "cityTimeCircle">
+      <input class ="cityTime" placeholder = "Time"></input>
     </section>
 
     <section class = "cityRectTall">
+      <input class ="cityCurrency" placeholder = "Currency"></input>
```
Comment (RGHANILOO) [inline public/city.html]: we have input here too we could probaly fix

**Review by RGHANILOO — CHANGES_REQUESTED** [review]
having also had a quick skim over css files, there are few areas that that styling are duplicated, for example font-family
my understanding of css (and i am not very good at it at all.) it cascades down so if you define font family on a parent the kids of that will also take on that styling.
over all if can also try and keep the mark up as semantic as possible it would be great

File: public/city.html (hunk 3 above)
Comment (tt01924) [reply public/city.html:18]: This is single day forecast currently - Spoke to Campbell my software engineer friend and he is going to think about the problem and hopefully get back to me with a solution.

**Pull request merged**